<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Strip Test Runner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableView" name="commandView">
       <property name="statusTip">
        <string>Sequence of commands to be sent to the Strip electronics</string>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextEdit" name="logMessageBrowser">
       <property name="statusTip">
        <string>Message pane</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="statusTip">
         <string>Progress status of the execution of the commands</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dryRunCheckBox">
        <property name="statusTip">
         <string>Do not send commands to the server, only simulate their execution</string>
        </property>
        <property name="text">
         <string>Dry run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runButton">
        <property name="statusTip">
         <string>Run the sequence of commands</string>
        </property>
        <property name="text">
         <string>&amp;Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runNextButton">
        <property name="statusTip">
         <string>Run the next available command and then stop</string>
        </property>
        <property name="text">
         <string>Run &amp;next command</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="logMessageLabel">
        <property name="text">
         <string>&amp;Log message:</string>
        </property>
        <property name="buddy">
         <cstring>logMessageEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="logMessageEdit">
        <property name="statusTip">
         <string>Text to be saved as a log message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addLogMessageButton">
        <property name="statusTip">
         <string>Save a new log message</string>
        </property>
        <property name="text">
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_load"/>
    <addaction name="action_save"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_Script">
    <property name="title">
     <string>Scri&amp;pt</string>
    </property>
    <addaction name="action_run"/>
    <addaction name="action_run_next_command"/>
    <addaction name="action_pause"/>
    <addaction name="action_stop"/>
    <addaction name="separator"/>
    <addaction name="action_reset_connection"/>
    <addaction name="separator"/>
    <addaction name="action_set_delay"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Script"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_load">
   <property name="text">
    <string>&amp;Load…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_run">
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="text">
    <string>&amp;Pause</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="action_run_next_command">
   <property name="text">
    <string>Run &amp;next command</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>&amp;Save as…</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_set_delay">
   <property name="text">
    <string>Set &amp;delay…</string>
   </property>
  </action>
  <action name="action_reset_connection">
   <property name="text">
    <string>Reset connection…</string>
   </property>
   <property name="toolTip">
    <string>Clear all the internal caches and re-establish a connection with the server</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>commandView</tabstop>
  <tabstop>logMessageBrowser</tabstop>
  <tabstop>dryRunCheckBox</tabstop>
  <tabstop>runButton</tabstop>
  <tabstop>runNextButton</tabstop>
  <tabstop>logMessageEdit</tabstop>
  <tabstop>addLogMessageButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
